test/pjstadig/macro.clj

```diff
@@ -0,0 +1,7 @@
+(ns pjstadig.macro)
```

Comment:
I accidentaly created this in same namespace as: src/pjstadig/macro.cljc

When I move it into pjstadig.fixtures.macro it clojurescript tests stops working (because of undefined functions)